apps/fxc-front/src/app/components/2d/planner-element.ts

```diff
@@ -261,4 +294,38 @@ export class PlannerElement extends connect(store)(LitElement) {
   private wheelSpeed(e: WheelEvent): void {
     store.dispatch(e.deltaY > 0 ? incrementSpeed() : decrementSpeed());
   }
+
+  private handleScoreAction() {
```

Comment:
It would be nice to display a modal while the track is being scored and allow cancelling the request.

You will probably need to add new message types for the worker, see https://www.typescriptlang.org/docs/handbook/unions-and-intersections.html#discriminating-unions

Reply:
I agree. Do you want we do that in this PR?

Reply:
For the moment I added a simple toast during the scoring, without any interruption mechanism.
My first attempt was to display a modal, but I find it's too big and didn't spent time to customize it with css.